app/public.js

```diff
@@ -70,13 +72,50 @@
         }
       }
     });
+    setEventListeners();
   }
 
   /**
+   * Refresh overlay
+   */
+  function refreshImagesInfo() {
+    hideImagesInfo();
+    showImagesInfo(null);
+  }
+
+  /**
+   * Set event listeners
+   */
+  function setEventListeners() {
+    let width = window.innerWidth;
+    let height = window.innerHeight;
+    let path = window.location.pathname;
+    let update;
+
+    interval = setInterval(function() {
```

Comment:
It's cleared in hideImagesInfo(). Assume we never call showImagesInfo() twice without hideImagesInfo()? Then we would get duplicate overlays. Could add a check in showImagesInfo() for isImagesInforActive() then hide first... new issue for that?